dm/dumpling/dumpling_test.go

```diff
@@ -124,3 +124,30 @@ func (d *testDumplingSuite) TestDefaultConfig(c *C) {
 	c.Assert(dumpling.dumpConfig.StatementSize, Not(Equals), export.UnspecifiedSize)
 	c.Assert(dumpling.dumpConfig.Rows, Not(Equals), export.UnspecifiedSize)
 }
+
+func (d *testDumplingSuite) TestCallStatus(c *C) {
+	dumpConf := export.DefaultConfig()
+	taskCfg := &config.SubTaskConfig{}
+	m := &Dumpling{cfg: taskCfg}
+	ctx := context.Background()
+
+	s := m.Status(nil).(*pb.DumpStatus)
+	c.Assert(s.CompletedTables, Equals, float64(0))
+	c.Assert(s.FinishedBytes, Equals, float64(0))
+	c.Assert(s.FinishedRows, Equals, float64(0))
+	c.Assert(s.EstimateTotalRows, Equals, float64(0))
+
+	dumpling, err := export.NewDumper(ctx, dumpConf)
+	c.Assert(err, NotNil)
+	m.core = dumpling
+
+	export.InitMetricsVector(dumpConf.Labels)
+
+	m.Close()
+
+	s = m.Status(nil).(*pb.DumpStatus)
+	c.Assert(s.CompletedTables, Equals, float64(0))
```

Comment:
can your explain the purpose of L149~L152?  what are these lines of code testing?

Reply:
thanks, I need more advice to fix this err. I think these lines of code can test whether the call of `func GetParameters()` is successful

Reply:
for the err in L141 , you can use `conn.InitMockDB(c)` to mock db Connection, you can see a example[ here](https://github.com/pingcap/ticdc/blob/f0a38a7aaf9f3b11a4d807da275b567642733f58/dm/dumpling/util_test.go#L35)

if you want to test `GetParameters/dumpling metrics` works as expected ,i thinks add a query-status it-test  is a better choice, maybe use dmctl query-status  to check metrics not equal to zero in this [full-mode-test](https://github.com/pingcap/ticdc/blob/f0a38a7aaf9f3b11a4d807da275b567642733f58/dm/tests/full_mode/run.sh#L133)

Reply:
Maybe you'd better use `&Dumper{}` for test.

Reply:
> for the err in L141 , you can use `conn.InitMockDB(c)` to mock db Connection, you can see a example[ here](https://github.com/pingcap/ticdc/blob/f0a38a7aaf9f3b11a4d807da275b567642733f58/dm/dumpling/util_test.go#L35)
> 
> if you want to test `GetParameters/dumpling metrics` works as expected ,i thinks add a query-status it-test is a better choice, maybe use dmctl query-status to check metrics not equal to zero in this [full-mode-test](https://github.com/pingcap/ticdc/blob/f0a38a7aaf9f3b11a4d807da275b567642733f58/dm/tests/full_mode/run.sh#L133)

thanks for your advice, I will add a it-tets for query-status in next PR(because I haven't run this it-test successfully)

Reply:
lgtm